setup.py

```diff
@@ -228,6 +228,11 @@ def use_bundled_luajit(path, macros):
     build_env = dict(os.environ)
     src_dir = os.path.join(path, "src")
     if platform.startswith('win'):
+        try:
+            subprocess.run(["C:\\Program Files(x86)\\Microsoft Visual Studio\\2019\\Community\\VC\\Auxiliary\\Build\\vcvarsall.bat", "x86_x64",])
```

Comment:
I don't think it helps to do this. IIUC, the batch file sets up environment variables, so it needs to run in the same command shell that then executes the build.

Also, I don't think it's a good idea to use a hard coded path. There must be a way to figure it out programmatically. It might also be that the batch file is simply on the search PATH already and can be started as is.

Note that appveyor isn't really used any more since we use GHA also for the Windows builds. I should disable it.